hazelcast-jet-core/src/main/java/com/hazelcast/jet/pipeline/WindowDefinition.java

```diff
@@ -69,11 +69,24 @@ public WindowDefinition setEarlyResultsPeriod(long earlyResultPeriod) {
 
     /**
      * Returns a sliding window definition with the given parameters.
+     * In a sliding window, the incoming items are grouped by fixed-size
+     * and overlapping intervals where the windows "slide" by the given size.
      * <p>
-     * Find more information in the Hazelcast Jet Reference Manual, Sliding and
-     * Tumbling Window.
+     * For example, given the timestamps
+     * {@code [0, 1, 2, 3, 4, 5, 6]}, {@code windowSize} of 4 and {@code @slideBy} of 2,
+     * the timestamps would be grouped into the following windows:
      *
-     * @param windowSize the size of the window (size of the range of the timestamps it covers)
+     * <pre>
+     *     [0, 1], [0, 1, 2, 3], [3, 4, 5, 6], [5, 6]
```

Comment:
Is it correct? Should not it be [0, 1], [0, 1, 2, 3], [2, 3, 4, 5], [4, 5, 6], [6] ?

Reply:
You are right, I rewrote this several times and still got it wrong :)